tests/repository_simulator.py

```diff
@@ -163,6 +163,15 @@ def add_signer(self, role: str, signer: SSlibSigner) -> None:
             self.signers[role] = {}
         self.signers[role][signer.key_dict["keyid"]] = signer
 
+    def rotate_keys(self, role: str) -> None:
+        """remove all keys for role, then add threshold of new keys"""
+        self.root.roles[role].keyids.clear()
+        self.signers[role].clear()
+        for _ in range(0, self.root.roles[role].threshold):
+            snapshot_key, snapshot_signer = self.create_key()
+            self.root.add_key(role, snapshot_key)
+            self.add_signer(role, snapshot_signer)
```

Comment:
We could use generic names here for the new keys and signer once it is valid for any role.
```suggestion
            key, signer = self.create_key()
            self.root.add_key(role, key)
            self.add_signer(role, signer)
```

Reply:
oh right, thanks